node/test/Test/Pos/Network/Windows/DnsDomainsSpec.hs

```diff
@@ -16,19 +16,19 @@
 -- | Specification of 'Pos.Network.Windows.DnsDomains'
 
 module Test.Pos.Network.Windows.DnsDomainsSpec
```

Comment:
I want @rockbmb or @adinapoli-iohk to review changes in this module.
Or just revert these changes, I don't understand why this PR touches this module.

Reply:
@gromakovsky from a quick eyeball it seems like they are all cosmetic changes to please whatever linter was in scope when writing this code 😉 . At glance, the changes:

* Moves the `if !defined(POSIX)` upper to include modules that are used only when running the spec on windows (otherwise a "redundant import" message would be shown);

* Moves the position of the `prop_GetWindowsDefDnsServer`;

I cannot check this on my machine as I don't use Windows, but surely AppVeyor should reject this code if we made a mistake, so it's 👍  on my part.

Reply:
Ok, @ksaric, then please just check that these tests are indeed launched by appveyor and were not accidentally disabled.

Reply:
@gromakovsky They were launched, and that's why they failed, that's why I updated the test. 
I hope it works this time.